source/prerequisites/my-sql.txt

```diff
@@ -10,4 +10,92 @@ MySQL Migration Prerequisites
    :depth: 1
    :class: singlecol
 
-MySQL.
\ No newline at end of file
+To run sync jobs from a MySQL source database, the database may require 
+some configuration changes. If Relational Migrator determines the 
+database needs configuration changes, it automatically generates a 
+SQL script with the required changes. It is recommended to have a 
+Database Administrator (DBA) review the commands in this script and 
+perform their execution on the database server. This topic 
+provides more details on the required configuration steps.
+
+.. include:: /includes/fact-short-sync-job-desc.rst
+
+For details on supported versions of MySQL, see 
+:ref:`supported-databases`.
+
+About this Task
+---------------
+
+Connecting Relational Migrator to MySQL 
+requires the `binary log <https://dev.mysql.com/doc/refman/8.0/en/binary-log.html>`__ 
+enabed on your MySQL instance. The binary log (Binlog) records all 
+operations in the order they are committed to the database. 
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: (Optional) User permissions setup 
+
+      Relational Migrator generates SQL that can be executed to enable 
+      ``Binlog`` for the source MySQL database. The following 
+      code creates a new MySQL service account for Relational Migrator 
+      to connect to the MySQL instance. 
+
+      Create a service account:
+
+      .. code-block:: sql
+         :copyable: true
+
+         CREATE USER 'user'@'localhost' IDENTIFIED BY 'password';
+
+      Grant the required permissions to the service account:
+
+      .. code-block:: sql
+         :copyable: true
+         
+         GRANT SELECT, RELOAD, SHOW DATABASES, REPLICATION SLAVE, REPLICATION CLIENT ON *.* TO 'user'@'%';
+
+      Apply the user privilege changes:
+
+      .. code-block:: sql
+         :copyable: true
+
+         FLUSH PRIVILEGES; 
+
+   .. step:: Manually verify Binlog is enabled
+
+         Relational Migrator automaticaly checks this setting for 
```

Comment:
[Question]

Do we need to manually check if it is already automatically checked by RM? If so, should we mark this step as optional? 

Also, what would be some reasons to check?

Reply:
I added this step as optional, you will need to check if binlog is enabled in order to run continuous jobs.